pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -1681,12 +1681,13 @@ func (m *kubeGenericRuntimeManager) GeneratePodStatus(event *runtimeapi.Containe
 	sort.Sort(containerStatusByCreated(kubeContainerStatuses))
 
 	return &kubecontainer.PodStatus{
-		ID:                kubetypes.UID(event.PodSandboxStatus.Metadata.Uid),
-		Name:              event.PodSandboxStatus.Metadata.Name,
-		Namespace:         event.PodSandboxStatus.Metadata.Namespace,
-		IPs:               podIPs,
-		SandboxStatuses:   []*runtimeapi.PodSandboxStatus{event.PodSandboxStatus},
-		ContainerStatuses: kubeContainerStatuses,
+		ID:                      kubetypes.UID(event.PodSandboxStatus.Metadata.Uid),
+		Name:                    event.PodSandboxStatus.Metadata.Name,
+		Namespace:               event.PodSandboxStatus.Metadata.Namespace,
+		IPs:                     podIPs,
+		SandboxStatuses:         []*runtimeapi.PodSandboxStatus{event.PodSandboxStatus},
+		ActiveContainerStatuses: kubeContainerStatuses,
```

Comment:
Do these kubeContainerStatuses only contain the container statuses of the current pod sandbox?